TeamID Team
bool in_air
telemetry telemetry
int64 telemetry_id
float64 telemetry_age_sec
string telemetry_timestamp